src/blog/2018-9-26/csharp-and-scala-support.pug

```diff
@@ -0,0 +1,57 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        ## Getting started
+
+        As you might have noticed, we've redesigned [our website](/). It is focussed to help you get started on the platform of choice. 
+
+        * [Get started with Stryker.NET](/stryker-net/quickstart)
+        * [Get started with Stryker4s](/stryker4s/quickstart)
+        * [Get started with Stryker](/stryker/quickstart) (for JavaScript / TypeScript)
+
+        ## New platforms
+
+        As you might know, both C# and Scala run on entirely different platforms compared to Stryker (which runs on NodeJS) does. This is the main reason we decided to implement them as
+        separate frameworks. [Stryker.NET](/stryker-net) is written in C# and runs as a .NET Core application, while [Stryker4s](/stryker4s) is written in Scala and runs on the JVM (Java Virtual Machine).
+        As of now, a version of Stryker.NET can be found on nuget.org, the package manager for the .NET platform. In due time, Stryker4s can be found on Maven Central.
+
+        Stryker.NET is the result of the internship of [Richard](https://github.com/richardwerkman)  
+        Stryker4s is the result of the internship of [Hugo](https://github.com/hugo-vrijswijk).
+
+        Thanks to them both for their hard work and continued support.
+
+        ## One design mentality
+        
+        Although all 3 versions of Stryker are implemented on different platforms, the design goals are the same. It should be _easy to use_ and
+        _fast to run_. Without compromises. 
+        
+        For Stryker JavaScript we're mutating _source code_ (as explained in our 
+        [road to Stryker 1.0](/blog/2017-07-14/road-to-stryker-1-0) blog post), instead of transpiled/minified JavaScript code. 
+        The main advantage is that it makes sure we only mutate _your actual code_, instead of 3rd party library or minified code.
+        
+        We wanted to offer the same quality to the .NET and Scala counterparts, but they have one additional problem: compiling code takes a long time.
+        It would dramatically reduce the performance of the frameworks. This is why we chose to implement a new mutation testing technique we call _Mutation Switching_.
+        With this technique, we compile only once. This results in a drastic performance gain, without compromising the quality of the mutations. 
+        It really deserves it's own blog post, so more on that later.
```

Comment:
Maybe here could be added that stryker (js) plans to introduce mutation switching too after the 1.0 release?

Reply:
I think we could leave that detail for that blogpost (#93). It already is a lot of details as it is INMHO.

Reply:
I'm playing with the idea to just delete this entire paragraph... it might be too detailed. Ideas?

Reply:
I think you can leave it in. It isn't that long and it is a nice link to the next blog post.

Reply:
Ok, I'll leave it in, thanks for the feedback